extensions/wikia/Recirculation/services/CakeRelatedContentService.php

```diff
@@ -141,13 +146,36 @@ private function formatTitle(Content $content) {
 
 		if ($content->getContentType() == "Discussion Thread") {
 			return $wgContLang->truncate(
-					"[Discussions] {$content->getTitle()}",
+					$content->getTitle(),
 					self::DISCUSSION_THREAD_TITLE_MAX_LENGTH);
 		}
 
 		return $content->getTitle();
 	}
 
+	private function getAuthor($metadata) {
+		if (array_key_exists("authorName", $metadata)) {
+			return $metadata["authorName"];
+		} else {
+			return "";
+		}
+
+	}
+
+	private function getRecirculationContentType($contentType) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)